src/Codeception/Module/Yii2.php

```diff
@@ -196,7 +199,9 @@ public function _before(TestInterface $test)
      */
     private function loadFixtures($test)
     {
-        if (method_exists($test, self::TEST_FIXTURES_METHOD)) {
+        if (empty($this->loadedFixtures)
```

Comment:
@samdark now i got a doubt. what if two cest classes define a `_fixtures()` method?

will the first one run but the second wouldn't? are all cest classes handled by the same instance of yii2 module?